tutorials/fundamentals/remove-last-char.md

```diff
@@ -0,0 +1,33 @@
+To remove the last charcter from a string in JavaScript, you should use the `slice()` method.
+It takes two arguments, what index to start and what index to end.
+`slice()` has negative indexing, which means that you could do `slice(0, -1)` and it would be equivalent to `slice(0, str.length - 1)`.
+
+```javascript
+let str = 'Masteringjs.ioF';
+str.slice(0, -1); // Masteringjs.io
+```
+
+## Alternative Methods
+
+`slice()` is generally easier, however other methods available are `substring()` and `replace()`.
+`substring()` does not have negative indexing, so be sure to use `str.length - 1` when removing the last character from the string.
+`replace()` takes either a string or a regular expression as its `pattern` argument but if it is a string, it only replaces the first instance.
```

Comment:
The "if it is a string" part about `replace()` isn't relevant here, because for the purposes of this tutorial we're only using regular expressions. Just explain that the regular expression `/.$/` matches the last character of the string, so `replace(/.$/, '')` replaces the last character of the string with an empty string